content/docs/esc/integrations/dynamic-secrets/gcp-secrets.md

```diff
@@ -1,16 +1,16 @@
 ---
-title_tag: gcp-secrets Pulumi ESC Provider
-meta_desc: The gcp-secrets Pulumi ESC Provider enables you to dynamically import Secrets from Google Cloud Secrets Manager into your Environment.
 title: gcp-secrets
+title_tag: gcp-secrets Pulumi ESC provider
+meta_desc: The gcp-secrets Pulumi ESC provider enables you to dynamically import secrets from Google Cloud Secrets Manager into your environment.
 h1: gcp-secrets
 meta_image: /images/docs/meta-images/docs-meta.png
 menu:
   pulumiesc:
     identifier: gcp-secrets
-    parent: esc-providers
-    weight: 6
+    parent: esc-dynamic-secrets
+    weight: 3
 aliases:
-- /docs/pulumi-cloud/esc/providers/gcp-secrets/
+    - /docs/pulumi-cloud/esc/providers/gcp-secrets/
```

Comment:
Missing an alias. (I'll stop commenting on these, but we should make sure every page that's been renamed has an alias that matches the path it's being moved from.)

Reply:
fixed